elastic/eui
# fix(EuiDualRange): Fix jumping to outer values when using empty string range

## Summary

This fixes #6853 - a bug where using any variation of empty string values (e.g. `['','']`) was causing the sliders to jump to their outer values when trying to move a slider over its half of the slider track. It was caused by incorrect parsing of empty strings in EuiDualRange resulting in `isValid = false` state and resetting to default values coming from `min` and `max` properties.

## QA

1. Open https://eui.elastic.co/pr_6858/#/forms/range-sliders#inputs
3. Find `With empty strings as the initial value` example
4. Click on the input and move the lower value slider from its default value to `76`
5. See it's behaving as expected

### General checklist

- [ ] Revert `[REVERT ME!]` temporary docs commit
